### DISCUSSION TITLE
Predictive/corrective text?

### DISCUSSION THREAD

**Discussion by TadDavisPhila** [vincode-io/Zavala#234] [Support] [answered]
I’m not sure what the right term for this is, but my typing on the phone is so clumsy that I rely heavily on the predictive text feature of the standard keyboard. I don’t see that option in Zavala or any way to turn it on. Am I missing something?

**Comment by vincode-io** [comment, marked as answer]
It looks like you have disabled Check Spelling While Typing and Correct Spelling Automatically. These settings are settable per Outline and you can change them in the Outline's Get Info dialog. If you want to automatically have these settings enabled for new Outlines, you can also change them in Settings.

**Comment by TadDavisPhila** [comment]
Thanks. I did have it turned on globally but somehow had missed that it was turned off on the outline I was working on.

—
Tad Davis
***@***.***
